src/main/com/intellij/lang/jsgraphql/ide/documentation/GraphQLDocumentationProvider.java

```diff
@@ -47,6 +33,12 @@
     private final static String GRAPHQL_DOC_PREFIX = "GraphQL";
     private final static String GRAPHQL_DOC_TYPE = "Type";
 
+    // using this instead of DocumentationMarkup to keep 173 compatibility
```

Comment:
I bumped the min version to 182 yesterday to ensure SVG icons are supported. Is this still needed then?

Reply:
Nope, it's better to replace them with DocumenationMarkup ones then